README.md

```diff
@@ -1799,6 +1799,124 @@ suggestions for consideration. The actual implementation is at the discretion of
 - `stream.Readable` &mdash; Either `@@dispose()` or `@@asyncDispose()` as an alias or [wrapper][] for `destroy()`.
 - ... and many others in `net`, `readline`, `tls`, `udp`, and `worker_threads`.
 
+# Best Practices
+
+## Ensuring Cleanup of Native Handles, File Descriptors, etc.
+
+When implementing a disposable that holds a native handle (i.e., a pointer, file descriptor, etc.), it is good practice
```

Comment:
Why only a good practice for native handles? If I were to implement a Disposable to let say removeEventListener() when disposed, and use it everywhere to replace addEventListener() calls, I would definitely want to make sure it can be disposed on GC in case someone forget to use it, and print out a warning if that happens.

Reply:
While that's a valid case, that leans more towards @bakkot's suggestion that such things be reserved for debug builds. Native handles are often a different story as such leaks can potentially cause system instability. Also, callbacks registered via `addEventListener` would be visible in a heap dump/snapshot which makes such leaks easier to find than a stray `Number` value.

Reply:
You need to make it a weakRef in that scenario, however EventTarget do not hold weakRefs.